### PR TITLE
Updating docs to reflect self creating storage pools

### PR DESCRIPTION
**Pull request by joedborg**

### CODE REVIEW

File: pages/k8s/storage.md [hunk 1]
```diff
@@ -76,7 +76,19 @@ juju config kubernetes-master allow-privileged=true
 
 ### Create storage pools
 
-Finally, the pools that are defined in the storage class can be created:
+By default, the `kubernetes-master` charm will create the required pools defined
+in the storage class.  If you're happy with this, you can skip the section.
```
Comment (hyperbolic2346) [inline pages/k8s/storage.md]: What are these defaults? How can the user see the settings?

File: pages/k8s/storage.md [hunk 2]
```diff
@@ -76,7 +76,19 @@ juju config kubernetes-master allow-privileged=true
...
+juju run --unit ceph-mon/0 "ceph tell mon.\* injectargs '--mon-allow-pool-delete=true'"
+
+juju run-action ceph-mon/0 delete-pool pool-name=xfs-pool --wait
+juju run-action ceph-mon/0 delete-pool pool-name=ext4-pool --wait
+```
+
+Then recreate them:
 
 ```bash
 juju run-action ceph-mon/0 create-pool name=xfs-pool --wait
```
Comment (hyperbolic2346) [inline pages/k8s/storage.md]: How would I specify a different number of replicas? What options might I want to change? What are the names of the pools generated?

File: pages/k8s/storage.md [hunk 3]
```diff
@@ -88,7 +93,8 @@ juju run-action ceph-mon/0 delete-pool pool-name=xfs-pool --wait
 juju run-action ceph-mon/0 delete-pool pool-name=ext4-pool --wait
 ```
 
-Then recreate them:
+Then recreate them, using the options listed from the `list-actions` command ran
+earlier.  For example:
```
Comment (hyperbolic2346) [inline pages/k8s/storage.md]: for example, no options! :-P

I hate to keep poking this on you, but could you change the run-action to use a different option like more replicas or something?

Comment (joedborg) [reply pages/k8s/storage.md]: No problem, measure twice; cut once and all that.  I was assuming that was explicit with what was above, but I'll add some changed args.

Comment (hyperbolic2346) [reply pages/k8s/storage.md]: I like it, it gives a user an idea of what they might not like about the defaults and shows them how to change the values around.

**Comment by hyperbolic2346** [comment]
I'm not comfortable merging this, I'll let @evilnick do it when he gets back from the holidays.

**Pull request merged**